examples/transform/src/main/java/org/apache/apex/examples/transform/DynamicTransformApplication.java

```diff
@@ -20,49 +20,29 @@
 package org.apache.apex.examples.transform;
 
 import java.util.Arrays;
-import java.util.HashMap;
-import java.util.Map;
 
 import org.apache.hadoop.conf.Configuration;
 
 import com.datatorrent.api.Context;
 import com.datatorrent.api.DAG;
 import com.datatorrent.api.StatsListener;
-import com.datatorrent.api.StreamingApplication;
 import com.datatorrent.api.annotation.ApplicationAnnotation;
-import com.datatorrent.lib.io.ConsoleOutputOperator;
 import com.datatorrent.lib.partitioner.StatelessThroughputBasedPartitioner;
 import com.datatorrent.lib.transform.TransformOperator;
 
 @ApplicationAnnotation(name = "DynamicTransformApp")
 /**
  * @since 3.7.0
  */
-public class DynamicTransformApplication implements StreamingApplication
+public class DynamicTransformApplication extends SimpleTransformApplication
 {
   private static String COOL_DOWN_MILLIS = "dt.cooldown";
   private static String MAX_THROUGHPUT = "dt.maxThroughput";
   private static String MIN_THROUGHPUT = "dt.minThroughput";
 
   @Override
-  public void populateDAG(DAG dag, Configuration conf)
+  void setPartitionerForTest(DAG dag,Configuration conf,TransformOperator transform)
```

Comment:
shouldn't this be setPartitioner - since it isn't for test purposes.

Reply:
corrected.